.github/renovate.json5

```
@@ -17,14 +17,13 @@
     {
       groupName: "all devDependencies",
       groupSlug: "all-dev",
-      matchPackagePatterns: ["*"],
       matchDepTypes: ["devDependencies"],
-      excludePackagePatterns: ["next"],
+      matchPackageNames: ["*", "!/next/"],
     },
     {
       groupName: "all @types",
       groupSlug: "all-types",
-      matchPackagePatterns: ["@types/*"],
+      matchPackageNames: ["/@types/*/"],
```

Not sure if this is correct, it might need to be 
```suggestion
      matchPackageNames: ["/@types/.*/"],
```
instead.

I'll check at a later date as this doesn't seem to be pressing.